src/Microsoft.Health.Dicom.Blob/Utilities/StoreConfigurationAware.cs

```diff
@@ -0,0 +1,23 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Blob.Utilities
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    internal abstract class StoreConfigurationAware : IStoreConfigurationAware
```

Comment:
Also, I find this naming a bit weird. Seems like it should be something like `BlobConfigurationSection`

Reply:
Sure will change it. @josiahvinson @bcarthic do you have any preference or thoughts on the naming? Otherwise will go with Brandon's suggestion.

Reply:
i am fine with BlobConfigurationSection

Reply:
Done